dist_thresh: .05